src/functions.php

```diff
@@ -26,3 +32,17 @@ function listClassesInDirectory(string $directory): iterable
         yield $class->getName();
     }
 }
+
+/**
+ * get a list of all classes in the given direcotory.
+ *
+ * Based on: https://github.com/Roave/BetterReflection/blob/396a07c9d276cb9ffba581b24b2dadbb542d542e/demo/parsing-whole-directory/example2.php.
```

Comment:
Could you remove this line? Only makes sense for the function above.

Reply:
sure

Reply:
Cheers :+1: